### PR TITLE
code cleanup based on PVS-Studio findings

### PR DESCRIPTION
**Pull request by inponomarev**
## Overview

Quick fixes based on PVS studio findings

---

### Checklist

- [ ] Change is covered by automated tests.
- [ ] JavaDoc is provided for all new/changed public APIs.
- [ ] User Guide is updated.
- [x] CI builds pass.

### CODE REVIEW

File: celesta-sql/src/main/java/ru/curs/celesta/score/ForeignKey.java [hunk 1]
```diff
@@ -196,12 +196,6 @@ void setReferencedTable(String grain, String table) throws ParseException {
 
         // Извлечение таблицы по имени.
         Table t = gm.getElement(table, Table.class);
-        if (t == null) {
```
Comment (inponomarev) [inline celesta-sql/src/main/java/ru/curs/celesta/score/ForeignKey.java:199]: `t` never equals `null`

File: celesta-sql/src/main/java/ru/curs/celesta/score/Index.java [hunk 2]
```diff
@@ -30,10 +30,6 @@ protected String getErrorMsg(String name) {
             throw new IllegalArgumentException();
         }
         table = getGrain().getElement(tableName, Table.class);
-        if (table == null) {
```
Comment (inponomarev) [inline celesta-sql/src/main/java/ru/curs/celesta/score/Index.java:33]: `table` never equals `null`

File: dbschemasync/src/main/java/ru/curs/celesta/dbschemasync/DBSchema2Celesta.java [hunk 3]
```diff
@@ -65,7 +65,7 @@ public static void dBSToScore(
 
                 Namespace namespace = extractNamespace(schemaName);
                 String grainName = extractGrainName(schemaName);
-                if (grainName == null || grainName.isEmpty()) {
```
Comment (inponomarev) [inline dbschemasync/src/main/java/ru/curs/celesta/dbschemasync/DBSchema2Celesta.java:68]: `grainName` never equals `null`

**Comment by packpaul** [comment]
@inponomarev What is PVS-Studio?

**Review by packpaul — APPROVED** [review]

**Comment by inponomarev** [comment]
@packpaul [PVS-Studio](https://www.viva64.com/en/pvs-studio/) is a proprietary static analysis tool. I'm testing it while preparing my conference talk about devops view on static analysis.

**Pull request merged**